src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -54,8 +54,29 @@ export class AppointmentsPage extends Component {
         }
       });
     } else {
-      // TODO: show no appointments message
-      content = 'No appointments';
+      content = (
```

Comment:
I copied this from the existing implementation that Narin had shared.

Alternatively we could use this, which I think is a little cleaner:

```
<li>
  <AlertBox
    className="vads-u-background-color--gray-lightest"
    backgroundOnly
  >
    <h2 className="usa-alert-heading vads-u-font-size--md">
      You don’t have any appointments.
    </h2>
    <p>
      You can schedule an appointment now, or you can call your{' '}
      <a
        href="/find-locations"
        target="_blank"
        rel="noopener noreferrer"
      >
        VA Medical center
      </a>{' '}
      to schedule an appointment.
    </p>
    <Link to="new-appointment">
      <button
        type="button"
        className="usa-button vads-u-margin-x--0 vads-u-margin-bottom--1p5"
        name="newAppointment"
      >
        Schedule an appointment
      </button>
    </Link>
  </AlertBox>
</li>
```